ansible/update_pw_python.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: all
+  tasks:
+    - include: update_pw_python.yml
+      become: true
+      become_user: root
+- name: Fetch updated piecewise repo
+  git: repo=https://github.com/opentechinstitute/piecewise.git 
+       dest=/opt/piecewise.git/
+       version=update-ndt-test
+- name: Deploy updated piecewise code
+  file: src=/opt/piecewise.git/{{ item.src }} dest=/opt/{{ item.dest }} state=link
```

Comment:
Not a big deal, but it would be nice if action parameters followed the following pattern, with the action on it's own line and each parameter indented below on its own line:

```
action:
  parameter1=something
  parameter2=another
  parameter3=else
```